broker/tests.py

```diff
@@ -0,0 +1,104 @@
+from broker.master import KolejkaCommunicationServer
+from http import client
+from threading import Thread, Lock
+import random as rand
+from time import sleep
+
+
+def test1_kolejka_communication(sub_number=1000, host='127.0.0.1', port=8080, max_wait=0.01) -> int:
```

Comment:
Trzeba przerobić to na klasę unittestów. Link do dokumentacji: [Unittesty w Pythonie](https://docs.python.org/3/library/unittest.html). Przykład tego jak to robić jest w oryginalnym repo BaCy2.